Small tweaks to DVP content (#17047)
docker/docs: docker-hub/publish/insights-analytics.md

following calendar month. You can import this data into your own systems, or you
can analyze it manually as a spreadsheet.

### Export data

> Export usage data for your organization's images using the Docker Hub website by following these steps:

Reviewer: Nit:

```suggestion
Export usage data for your organization's images from Docker Hub by following these steps:
```

pulls. To provide feedback or ask questions about these rules,
[fill out the Google Form](https://forms.gle/nb7beTUQz9wzXy1b6){:
target="_blank" rel="noopener" class="_"}.

> <!-- TODO: What do the first columns mean? -->

Reviewer 2: iirc: the first column is the HTTP method, the second column is the image reference type (image tag name vs SHA 256 digest)

| HEAD           | tag       | N/A                                                             | Version check    | User already has all layers existing on local machine                                                          | This is similar to the use case of a pull by tag when the user already has all the image layers existing locally, however, it differentiates the user intent and classifies accordingly.                                                                                              |
| GET            | tag       | N/A                                                             | Pull by tag      | User already has all layers existing on local machine and/or the image is single-architecture                          |

> | GET            | tag       | Get by different digest                                         | Pull by tag      | Image is multi-architecture                                                                                            | Second GET by digest must be different from the first.                                                                                                                                                                                                                                         |

Reviewer: the term 'multi-arch' is very commonly used, I'm not sure we should switch to the abbreviated form (and therefore also keep "single-arch" for consistency)

Author: Awwright then, gosh darn that jargon ;)
